docs/making-changes.rst

```diff
@@ -0,0 +1,34 @@
+Making changes
+==============
+
+Salt changes - Testing against a virtual machine or making changes against live servers
+---------------------------------------------------------------------------------------
+
+In other code repositories, we require work to be done in a branch and tested on local machines before a pull request is made. Approval is needed before merging.
+
+Sometimes, it is possible to test changes to these scripts against a virtual machine and in a separate branch.
```

Comment:
We should outline how one does this. Happy for that comment to live in an issue at this stage.

Reply:
Yes, that could get very complex. Let's issue it.